packages/eslint-config/.eslintrc.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
-  extends: ["react-app", "airbnb", "prettier"],
-  plugins: ["prettier"],
```

Comment:
I was looking at the docs for this and it says it
> Runs Prettier as an ESLint rule and reports differences as individual ESLint issues.
which means it's not doing anything, since we run Prettier as a commit hook. Removed.